lib/elements/custom-style.html

```diff
@@ -19,17 +19,43 @@
 
   /**
    * Custom element for defining styles in the main document that can take
-   * advantage of several special features of Polymer's styling system:
+   * advantage of [shady DOM](https://github.com/webcomponents/shadycss) shims
+   * for style encapsulation, custom properties, and custom mixins.
    *
    * - Document styles defined in a custom-style are shimmed to ensure they
    *   do not leak into local DOM when running on browsers without native
    *   Shadow DOM.
-   * - Custom properties used by Polymer's shim for cross-scope styling may
-   *   be defined in an custom-style. Use the :root selector to define custom
-   *   properties that apply to all custom elements.
+   * - Custom properties can be defined in an custom-style. Use the `html` selector 
```

Comment:
>Custom properties can be defined in an custom-style.

Should be 

Custom properties can be defined in a custom-style.

Also.. sometimes it is custom-style and other times `<custom-style>`. I am sure there is a reason, just wondering what it is :)